extra-security/clipass/autobuild/build

```diff
@@ -0,0 +1,2 @@
+echo "PKGDIR: $PKGDIR"
```

Comment:
Please use a verb-prefixed sentence to describe what is currently being done, for instance:

```
abinfo 'Installing clipass to $PKGDIR ...'
```

Note that I am using single quotation marks here, so that you won't be displaying a long `$PKGDIR`, not to mention that it changes from every build, making it quite pointless. Or really, keep it even shorter...

```
abinfo 'Installing clipass ...
```

Reply:
It was for debugging purposes. Now removed.